webapp/src/component/layout/TopBar/Notifications.tsx

```diff
@@ -63,7 +64,12 @@ export const Notifications: FunctionComponent<{ className?: string }> = () => {
         onClick={handleOpen}
         size="large"
       >
-        <Bell01 />
+        <Badge
+          badgeContent={notifications?.page?.totalElements}
+          color="secondary"
```

Comment:
If you need some specific color of the badge, you can use the `sx` param or wrap the Badge component with `styled` function and you can override the styles as you want.